app/src/main/java/tttTDD/Java/ConsoleClient.java

```diff
@@ -144,13 +145,43 @@ private String formatCellDisplay(String cell, String maxInput) {
         StringBuilder formattedCell = new StringBuilder();
 
         if (cell.length() <= maxInput.length()) {
-            StringBuilder spaceAlignment = new StringBuilder();
 
-            IntStream.range(0, maxInput.length() - cell.length()).forEach(i -> spaceAlignment.append(" "));
-            formattedCell.append(spaceAlignment);
+            IntStream.range(0, maxInput.length() - cell.length()).forEach(i -> formattedCell.append(" "));
             formattedCell.append(cell);
         }
 
         return formattedCell.toString();
     }
+
+    private int playerInput() {
+        try {
+            int value = input.nextInt();
+
+            if (value > 2) {
+                return value;
+            } else {
+                throw new Exception();
+            }
```

Comment:
Does this not make inappropriately constrain the human player's moves?

Reply:
More generally, I like the idea of a wrapper for the `Scanner` which forces us to handle the non-numeric exception, but it doesn't seem like this shared method should be responsible for handling whether it's the wrong _kind_ of number.

Reply:
After committing it I realized the same thing so I ended making a separate method for the `boardSize` input.